moveit_reach_plugins/include/moveit_reach_plugins/evaluation/manipulability_moveit.h

```diff
@@ -51,11 +51,24 @@ class ManipulabilityMoveIt : public reach::plugins::EvaluationBase
   std::vector<int> jacobian_row_subset_;
 };
 
-class ManipulabilityRatio : public ManipulabilityMoveIt
+class ManipulabilityNormalized : public ManipulabilityMoveIt
```

Comment:
I think we should add a little documentation about what it means to be normalized (like the text in the `plugin_description.xml`

```suggestion
/** @brief Computes the manipulability of a robot pose divided by the characteristic length of the robot */
class ManipulabilityNormalized : public ManipulabilityMoveIt
```